### PR TITLE
Update active tag tab containing edited tag

### LINKED ISSUE #6061: Tagtabs do not update when using Edit-Tags or opbase-less Tag-Equip/Invent
1) select a tag tab
2) Edit tags, removing the current tag-tab's tag
3) Observe the tab not updating until you reselect it

This also happens with the Opbase-less `Tag-Equipment` and `Tag-Inventory`

### PR DESCRIPTION
**Pull request by deathbeam**
Closes #6061

Signed-off-by: Tomas Slusny <slusnucky@gmail.com>

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/TabInterface.java [hunk 1]
```diff
@@ -456,12 +460,14 @@ else if (event.getMenuAction() == MenuAction.RUNELITE
...
 
-					final List<String> tags = SPLITTER.splitToList(newTags);
+					final List<String> tags = SPLITTER.splitToList(newTags.toLowerCase());
 
 					for (Integer item : items)
 					{
 						tagManager.addTags(item, tags);
 					}
+
+					checkActiveTab(tags);
```
Comment (raiyni) [inline runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/TabInterface.java]: This is unnecessary.

**Comment by deathbeam** [comment]
actually this is wrong, lemme fix

**Comment by deathbeam** [comment]
Now its correct

File: runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/TabInterface.java [hunk 2]
```diff
@@ -598,6 +604,14 @@ else if (event.getMenuAction() == MenuAction.RUNELITE
 		}
 	}
 
+	public void checkActiveTab(final Collection<String> tags)
```
Comment (Abextm) [inline runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/TabInterface.java]: `updateTabIfActive`?

**Comment by deathbeam** [comment]
updated

**Comment by Abextm** [comment]
`Tag-Inventory` and `Tag-Equipment` do nothing on this branch

**Comment by deathbeam** [comment]
Now they do (target is empty not null when none is set, or maybe one or either, so changed the condition to Strings.isNullOrEmpty)

**Pull request merged**